test/e2e/framework/test_context.go

```diff
@@ -320,6 +325,7 @@ func RegisterClusterFlags() {
 	flag.StringVar(&TestContext.EtcdUpgradeVersion, "etcd-upgrade-version", "", "The etcd binary version to upgrade to (e.g., '3.0.14', '2.3.7') if doing an etcd upgrade test.")
 	flag.StringVar(&TestContext.GCEUpgradeScript, "gce-upgrade-script", "", "Script to use to upgrade a GCE cluster.")
 	flag.BoolVar(&TestContext.CleanStart, "clean-start", false, "If true, purge all namespaces except default and system before running tests. This serves to Cleanup test namespaces from failed/interrupted e2e runs in a long-lived cluster.")
+	flag.BoolVar(&TestContext.SkipStartupWait, "start-now", false, "If true, disables multiple checks at the start of the test run ensuring nodes/pods/etc are in the proper state.")
```

Comment:
this needs to be explicit about what checks and other flags it makes us ignore (`--minStartupPods`, `--system-pods-startup-timeout`, `--system-daemonsets-startup-timeout`, `--node-schedulable-timeout`, `--allowed-not-ready-nodes`?)

Reply:
and the help for those other flags should be updated to add something like `This flag is ignored if --start-now is set.`

Reply:
would a name like `--skip-cluster-readiness-checks` be clearer?

Reply:
suggest aligning the testcontext var name with the flag (e.g. `SkipClusterReadinessChecks`)

Reply:
👍 all sound like good ideas. I don’t need to be worried about being too verbose in these flag descriptions?  If not, I’ll make those changes. I like the rename too, I struggled to find the right name.

Note though: I think that flag name makes sense if we are doing it at startup; how do you think that plays into the discussion above about whether or not that flag should be respected only at startup or every time the WaitFor... method is called?